### ISSUE TITLE
Mr. Mine (1397920)

### ISSUE THREAD

**Issue by 4164696f73** [ValveSoftware/Proton#4925]
# Compatibility Report
- Name of the game with compatibility issues: Mr. Mine
- Steam AppID of the game: 1397920

## System Information
- GPU: Nvidia GeForce 750 Ti
- Driver/LLVM version: Nvidia 465.31
- Kernel version: 5.10.44-1 (Linux-LTS)
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/4164696f73/34886f534376facc042f616c11f384ef
- Proton version: 6.9-GE-1

## I confirm:
- [x ] that I haven't found an existing compatibility report for this game.
- [x ] that I have checked whether there are updates for my system available.

[steam-1397920.log](https://github.com/ValveSoftware/Proton/files/6721188/steam-1397920.log)

## Symptoms
After attaching Proton to the game, the game never launches again and swapping/removing Proton from compatibility layer doesn't help at all.

## Reproduction
Set the compatibility tools to any Proton you want, then launch the game and it won't run.
The game is free, so you can even do it yourself. Simply install the game and launch it to see that it works, then add compatibility with Proton and launch it again to see that it no longer works. You can now remove the compatibility with Proton, but the game still won't launch.

Edit: Actually managed to make it work with Proton 6.3.5. I simply swapped it to that one and it worked. Took me literally 1 hour to figure out, but actually someone in the Steam forums helped me to find out Steam uses a default proton to run games. I actually didn't know that.
Edit2: Though, the framerate seems quite poor. Not unplayable, but you can see that it loses frames sometimes. I assume there was an update to Proton yesterday because two days ago (25th June) it worked smoothly.

**Comment by 4164696f73** [comment]
After recent Mr. Mine update, the game no longer works - it opens and closes itself immediately. It used to run only on 6.3.x, now it doesn't work at all (made sure to check all official Protons this time, unlike last time...). I assume it's nothing with Proton as the game worked normally, then I took a break for a month and wanted to come back today, and it doesn't want to launch. There were two updates to it recently (one major, one hotfix, or maybe more than one hotfixes).

System fully updated, kernel 5.10.88-2 LTS, everything else is the same as first post (and whatever the newest Nvidia drivers are). As mentioned, tested only and only with official Protons.
Log: [steam-1397920.log](https://github.com/ValveSoftware/Proton/files/7801021/steam-1397920.log)

**Comment by kisak-valve** [comment]
Hello @4164696f73, these look like lines of interest from your log:
```
err:vulkan:__wine_create_vk_instance_with_callback Failed to create instance, res=-1
[...]
err:   DxvkInstance::createInstance: Failed to create Vulkan 1.1 instance
[...]
err:   D3D11CreateDevice: Failed to create a DXGI factory
```
I'm seeing DXVK trying to start up in your log, but no hints that it found your video driver. Please share an updated gist of your system information from Steam with the extended diagnostics information.

**Comment by 4164696f73** [comment]
https://gist.github.com/4164696f73/d6b1297f804d1b1f72c5414cd181368f

Now that you mention it, I did have some weird issues after recent Nvidia updates, for example videos (only noticed in Youtube) were stuttering sometimes. Wouldn't be surprised if that would be on Nvidia's side.

**Comment by kisak-valve** [comment]
Yes, Vulkan is broken on your system with https://gist.github.com/4164696f73/d6b1297f804d1b1f72c5414cd181368f#file-gistfile1-txt-L159-L172 and https://gist.github.com/4164696f73/d6b1297f804d1b1f72c5414cd181368f#file-gistfile1-txt-L302-L315 as the sections to focus on.

If the system is otherwise healthy, I wouldn't be surprised if this specific situation is AMDVLK breaking other Vulkan drivers again. We've seen this before at https://github.com/GPUOpen-Drivers/AMDVLK/issues/187.

I'd test if removing the `amdvlk` and `lib32-amdvlk` system packages helps.

**Comment by 4164696f73** [comment]
Removing those fixed it. I actually didn't know what they were, and never bothered checking (as it was installed by default on Garuda), and I do, in fact, remember them being updated a few days ago (maybe yesterday or two days ago, don't remember).

Thank you! And yeah, my system is perfectly healthy aside of that.
